about_first_project.css

```diff
@@ -0,0 +1,122 @@
+body {
+    background-color: mintcream;
+    margin: 0px; 
+    color: black;
+}
```

Comment:
Matter of taste, but I think it's more readable when you set the tab size to be only 2 spaces (instead of 4 like you have):
```
body {
  background-color: mintcream;
  margin: 0px; 
  color: black;
}
```